erpnext/hr/doctype/upload_attendance/upload_attendance.py

```diff
@@ -114,6 +115,11 @@ def upload():
 		if not row: continue
 		row_idx = i + 5
 		d = frappe._dict(zip(columns, row))
+
+		if (d.get("name") and
+			frappe.db.get_value('Attendance', d.get('name'), 'docstatus') == 1):
```

Comment:
this should throw an error if the attendance value is different

Reply:
Fixed!
Have removed this code and added except AttributeError: pass to avoid the update_after_submit_error

Before 
![screen shot 2018-10-01 at 12 30 09 pm](https://user-images.githubusercontent.com/8780500/46274192-36bd7980-c576-11e8-991e-d1413c867d56.png)


After
![screen shot 2018-10-01 at 12 30 42 pm](https://user-images.githubusercontent.com/8780500/46274198-3a510080-c576-11e8-8881-75f90282327d.png)